camel/storages/vectordb_storages/milvus.py

```diff
@@ -118,40 +118,46 @@ def _create_collection(
 
         Args:
             collection_name (str): Name of the collection to be created.
-            **kwargs (Any): Additional keyword arguments.
+            **kwargs (Any): Additional keyword arguments pass to create
+                collection.
         """
-        from pymilvus import CollectionSchema, DataType, FieldSchema
-
-        index_params = {
-            "metric_type": "COSINE",
-            "index_type": "AUTOINDEX",
-            "params": {},
-        }
 
-        fields = [
-            FieldSchema(name='id', dtype=DataType.VARCHAR,
-                        descrition='A unique identifier for the vector',
-                        is_primary=True, auto_id=False, max_length=65535),
-            # limit from Milvus is 65535 https://milvus.io/docs/limitations.md
-            FieldSchema(
-                name='payload', dtype=DataType.JSON,
-                description=('Any additional metadata or information related'
-                             'to the vector')),
-            FieldSchema(
-                name='vector', dtype=DataType.FLOAT_VECTOR,
-                description='The numerical representation of the vector',
-                dim=self.vector_dim)
-        ]
-        schema = CollectionSchema(fields=fields,
-                                  description='Collection schema')
-
-        self._client.create_collection_with_schema(
+        from pymilvus import DataType
+
+        # set schema
+        schema = self._client.create_schema(auto_id=False,
+                                            enable_dynamic_field=True,
+                                            description='collection schema')
+
+        schema.add_field(field_name="id", datatype=DataType.VARCHAR,
+                         descrition='A unique identifier for the vector',
+                         is_primary=True, max_length=65535)
+        schema.add_field(
```

Comment:
Do you think it's necessary to provide the ref for the number '65535'? (`https://milvus.io/docs/limitations.md`)

Reply:
thanks for the review, added reference